operator/go.mod

```diff
@@ -12,10 +12,10 @@ require (
 	github.com/prometheus/client_golang v1.12.1
 	github.com/prometheus/common v0.32.1
 	github.com/stretchr/testify v1.7.1
-	k8s.io/api v0.24.0
-	k8s.io/apimachinery v0.24.0
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Is this the right client-go here?

Reply:
yeah, that's because of this dependency: `github.com/grafana/loki v1.6.2-0.20220420044148-f62b4ae1905c`

Reply:
Maybe we're a go for updating the Loki dependency?